doc/litespeed-configuration.rst

```diff
@@ -0,0 +1,137 @@
+same set of features (except for OLS not supporting Edge Side Includes (ESI)).
+
+So before you start configuring the server, make sure you know which version of LiteSpeed you are using.
+
+.. note::
+
+    Any LiteSpeed setup works in a single node web server environment only. If you are targeting a multi
+    node setup you might want to consider switching to :ref:`Varnish <varnish configuration>` which has excellent
+    support for this already built-in in this library.
+
+
+Configuring OpenLiteSpeed
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+OLS does not support different caching settings depending on ``.htaccess`` settings and different paths.
+If you need that, you have to go with LSWS instead.
+Thus, OLS has to be configured as follows on server or vHost level::
+
+    module cache {
+      # This enables the public cache
+      enableCache                      1
+
+      # This disables the private cache
+      enablePrivateCache               0
+
+      # This enables the public cache
+      checkPublicCache                 1
+
+      # This disables the private cache
+      checkPrivateCache                0
+
+      # Also consider the query string in caches
+      qsCache                          1
+
+      # Enable checking for a cached entry if there's a cookie on the request
+      reqCookieCache                   1
+
+      # We ignore request Cache-Control headers
+      ignoreReqCacheCtrl               1
+
+      # Must be disabled, this tells LS to check the Cache-Control/Expire headers on the response
+      ignoreRespCacheCtrl              0
+
+      # We don't cache responses that set a cookie
+      respCookieCache                  0
+
+      # Configure the maximum stale age to a sensible value for your application
+      # The maxStaleAge defines a grace period in which LS can use an out of date (stale) response while checking on a new version
+      maxStaleAge                      10
+
+      # Make sure we disable expireInSeconds because it would override our Cache-Control header
+      expireInSeconds                  0
+
+      # Enable the module
+      ls_enabled                       1
+    }
+
+That's all you need. Of course you might want to adjust certain values to your needs.
+Also refer to the OLS docs if you need more details about the different configuration values.
+
+Configuring LiteSpeed WebServer
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+LSWS can also be configured on server level, however, as LSWS supports ``.htaccess`` you may want to configure it
+there to give your application the flexibility of having different configurations for certain paths.
+
+Configure your ``.htaccess`` as follows::
+
+    <IfModule LiteSpeed>
+        CacheEnable public /
+        # TODO: The rest of the directives
+    </IfModule>
+
+Also refer to the LSWS docs if you need more details about the different configuration values.
+
+Configuring the library
```

Comment:
i'd say "Configuring FOSHttpCache to work with LiteSpeed" - library is a bit vague for the context switch

Reply:
Ok will do.